src/main/java/uk/gov/hmcts/reform/et/syaapi/models/ClaimantApplicationRequest.java

```diff
@@ -13,9 +13,10 @@ public class ClaimantApplicationRequest {
 
     @JsonProperty("case_id")
     private String caseId;
-
     @JsonProperty("case_type_id")
     private String caseTypeId;
+    @JsonProperty("is_type_c")
+    private boolean isTypeC;
```

Comment:
could be private boolean typeC, then the getter will be generated as 
`public boolean isTypeC() {
        return typeC;
    }`